var/spack/repos/builtin/packages/py-segmentation-models-pytorch/package.py

```diff
@@ -13,7 +13,9 @@ class PySegmentationModelsPytorch(PythonPackage):
     pypi = "segmentation_models_pytorch/segmentation_models_pytorch-0.2.0.tar.gz"
 
     license("MIT")
+    maintainers("adamjstewart")
```

Comment:
@qubvel @isaaccorley let me know if you would also like to maintain this build recipe.